_posts/2017-09-12-repeatable-serverless-applications.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Repeatable Serverless Applications"
+date: 2017-09-11
+categories: serverless architecture docker aws
+author: Rick Foxcroft
+image: img/no_servers.png
+---
+
+Building and testing serverless applications locally can be challenging. It's a relatively new landscape and there aren't many established practices, regarding good serverless application design.
+At a recent Mechanical Rock mob programming session we decided to tackle the topic of, "How can we turn off any superfluous and idle ec2 instances" that we may have forgotten to stop/terminate – because servers are expensive right? We decided to call the new project, Instance Reaper. Source code available <a href="https://github.com/MechanicalRock/instance-reaper">here</a>. 
```

Comment:
link to the source should be at the end - a carrot for those who've read that far.